_drafts/2017-07-27-taxonomy-suite.md

```diff
@@ -130,7 +131,7 @@ head(rbind(clazz))
 
```

Comment:
For `taxize` text, "...taxonomic data sources online, including NCBI, ITIS, GBIF, EOL, IUCN,..." would be good to link to some of these resources, otherwise not clear to non-expert what their value is.

In reference to Chamberlain, S. A., & Szöcs, E. (2013). taxize: taxonomic search and retrieval in R. F1000Research. better to link that text to the paper rather than showing the DOI URL separately below

Saying "about a month ago" or similar phrases in the post aren't helpful  - better to add month, year(?) 'cause people often refer back to these posts much later.

"You can now get downstream taxa ..." sounds jargony. Do you mean nested taxa? (still jargony). 

Each time you give example " of the power of " package, better to include a short statement on what the example is going to show. The "power" is hard to glean unless you're already in the know. For taxize, maybe say you can go from genus, species to full classification (or something)